tsconfig.json

```diff
@@ -26,5 +26,5 @@
     "target": "ESNext",
     "verbatimModuleSyntax": true
   },
-  "exclude": ["SampleApp/", "integration-testing/"]
+  "exclude": ["integration-testing/"]
```

Comment:
Here too. I feel like we just want to replace SampleApp with example.

Reply:
Same as the above comment -- it shouldn't be added here as we do want it to be type checked.  `integration-testing` is in here as it is going to be deleted later, so I'm not going to spend time typing it.